packages/lesswrong/components/comments/CommentsListSection.tsx

```diff
@@ -182,7 +184,7 @@ const CommentsListSection = ({
       >
         {commentSortNode}
       </Typography>
-      {post && <Typography
+      {post && !hideDateHighlighting && <Typography
```

Comment:
This looks like a change that'd affect other uses of `CommentsListSection`, since the default is undefined and will now hide everything under here.